bazel/hermetic_cc_toolchain.patch

```diff
@@ -6,19 +6,37 @@
 # Strip debug_info off every target for determinism
 # (https://github.com/ziglang/zig/issues/23821, https://github.com/ziglang/zig/issues/23823).
 diff --git a/MODULE.bazel b/MODULE.bazel
-index 409d626..1265298 100644
+index 409d626..15b70b3 100644
 --- a/MODULE.bazel
 +++ b/MODULE.bazel
-@@ -49,6 +49,7 @@ register_toolchains(
+@@ -49,7 +49,25 @@ register_toolchains(
      "@zig_sdk//libc_aware/toolchain:linux_arm64_musl",
      # wasm/wasi toolchains
      "@zig_sdk//toolchain:wasip1_wasm",
 +    "@zig_sdk//toolchain:none_wasm",
-
+ 
      # These toolchains are only registered locally.
      dev_dependency = True,
+ )
++
++bazel_dep(name = "bazel_skylib", version = "1.7.1")
++
++new_local_repository = use_repo_rule("@bazel_tools//tools/build_defs/repo:local.bzl", "new_local_repository")
++new_local_repository(
++    name = "strip",
++    build_file_content = """
++load("@bazel_skylib//rules:native_binary.bzl", "native_binary")
++
++native_binary(
++name = "strip_bin",
++src = ":strip",
++visibility = ["//visibility:public"],
++)
++""",
++    path = "/usr/bin",
```

Comment:
IIUC this (1) makes the whole `/usr/bin/` available as a repository, though without creating any dependency on it; and then `native_binary` is the one that actually creates a dependency. I'm still a bit confused though; how does it know to look for an exe called `strip` in there?

Reply:
Ha, got it!

```
src = ":strip",
```

this actually references the `strip` file, since it's exported from the repository